.gitignore

```diff
@@ -19,14 +19,14 @@ export/
 /runtime/release
 /cli/pkg/web/embed/dist
 /rill
+/cli/pkg/examples/embed/dist
 
 # data files
 *.duckdb
 *.duckdb.wal
 *.db
 *.wal
 *.parquet
-*.csv
```

Comment:
We used to have some random csvs on the root. So better to keep it and force add the ones we need to add.

Reply:
Yeah make sense, Removed as we were pushing the data files as well, reverted back